ppa/archive/templates/archive/snippets/search_result.html

```diff
@@ -1,11 +1,24 @@
-{% load static humanize %}
+{% load static humanize ppa_tags %}
 
 <li class="item {{ item.work_type }}">
-    {% if item.work_type == "full-work" %} {# unapi metadata only available for full works #}
-    <abbr class="unapi-id" title="{{ item.group_id }}" aria-hidden="true"></abbr>
+
+    {# Generate URL once for use in both COinS and main link #}
+    {% if item.first_page %}
+        {% url 'archive:detail' source_id=item.source_id start_page=item.first_page as detail_url %}
+    {% else %}
+        {% url 'archive:detail' item.source_id as detail_url %}
+    {% endif %}
```

Comment:
nice; took me a moment to remember why these differ, suggesting a comment to as a reminder
```suggestion
    {# Generate URL once for use in both COinS and main link #}
    {% if item.first_page %} {# generating detail url for excerpts requires specifying start page #}
        {% url 'archive:detail' source_id=item.source_id start_page=item.first_page as detail_url %}
    {% else %}
        {% url 'archive:detail' item.source_id as detail_url %}
    {% endif %}
```